src/cmd/package.go

```diff
@@ -75,18 +75,18 @@ var packageDeployCmd = &cobra.Command{
 	Aliases: []string{"d"},
 	Short:   lang.CmdPackageDeployShort,
 	Long:    lang.CmdPackageDeployLong,
-	Args:    cobra.MaximumNArgs(1),
+	Args:    cobra.ExactArgs(1),
```

Comment:
```suggestion
	Args:    cobra.MaximumNArgs(1),
```

`zarf package deploy` is a thing (with no args)